### DISCUSSION TITLE
Last version crash

### DISCUSSION THREAD

**Discussion by atomizasser** [Codectory/AutoActions#13] [Q&A]
Dear.
I've updated to last realease and after launch Project Cars 3, i get a error and game crash.
If i downgraded to version 1.5.7 (with the same configuration, overwriting files), the game will launch.
I've seen that last version is not x64 and run in 32bit mode.

**Comment by Codectory** [comment]
Should be fixed now

**Reply by atomizasser** [reply]
Thanks! I will try it later!
